HLTriggerOffline/Common/python/HLTValidation_cff.py

```diff
@@ -22,9 +22,9 @@
 from Validation.HGCalValidation.hgcalHitCalibrationDefault_cfi import hgcalHitCalibrationDefault as _hgcalHitCalibrationDefault
 hgcalHitCalibrationHLT = _hgcalHitCalibrationDefault.clone()
 hgcalHitCalibrationHLT.folder = "HGCalHitCalibrationHLT"
```

Comment:
@cms-sw/hcal-dpg-l2 @rovere, this apparently entered via https://github.com/cms-sw/cmssw/pull/44558. 
I would recommend to use `HLT/HGCalHitCalibration` instead (and not create one more top level folder), but I would defer it to the next PR.